sonar-xml-plugin/src/test/resources/checks/HardcodedCredentialsCheck/android_password.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android" xmlns:app="http://schemas.android.com/apk/res-auto" android:layout_width="match_parent" android:layout_height="match_parent">
+  <ImageButton android:id="@+id/assistance_button" android:background="@null" android:layout_width="wrap_content" android:layout_height="wrap_content" android:layout_margin="@dimen/default_margin" android:src="@drawable/ic_actionbar_help" android:layout_alignParentRight="true"/>
+  <com.aaa.views.FontEditText android:id="@+id/edit_password_0" android:hint="@string/hint_password" android:password="true"  app:font="gotham-book-clean.ttf" style="@style/LoginEditText"/>
+  <com.aaa.views.FontEditText android:id="@+id/edit_password_1" android:hint="@string/hint_password" android:password="false" app:font="gotham-book-clean.ttf" style="@style/LoginEditText"/> <!-- Noncompliant {{"password" detected here, make sure this is not a hard-coded credential.}} -->
+</RelativeLayout>
```

Comment:
There is a lot of noise in this test file, it is hard to understand what we are testing. We can probably reduce it to keep less code, even if the code would not make sense in an app.